src/fetchtastic/download/client_app.py

```
@@ -0,0 +1,1129 @@
+
+- app/<version>/
+- app/prerelease/<version>/
+"""
+
+import fnmatch
+import json
+import os
+import shutil
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import Any, Dict, List, Optional
```

Ruff (0.15.12)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)